UI/src/view/navigationpanel/massTransitStopBox.js

```diff
@@ -120,6 +120,8 @@
       return roadTypePanel.concat(constructionTypePanel).concat(massTransitStopPanel).concat(pointAssetTypePanel);
     };
 
+    this.constructionTypeLabeling = function () {};
```

Comment:
jo ylikirjoitus korjaa ongelman niin missä tää sitten määritellään täällä? 

Vähän mietin eikös oikea ratkaisu oo poistaa constructionTypeLabeling täällä niin että se hyödyntää perinnästä saatua constructionTypeLabeling muodostusta.

Reply:
No mun käsittääkseni tuolla this.template (rv. 245). Siellä on sit me.elements, joka määritellään yliluokassa, jonka osa toi constructionTypeLabeling on. Perinnästä saatu tuo siis labelin, joka on ylimääräinen täällä, mutta tarpeellinen monessa muussa legendissä.

Reply:
ok